scripts/dist.ts

```diff
@@ -123,25 +123,65 @@ function toDist(sourcePath: string): YAML.Document {
           `${id}: contains at least one deprecated compat feature and can never be Baseline. Was this intentional?`,
         );
       }
-      insertStatus(dist, JSON.parse(status.toJSON()));
+      const tmp = JSON.parse(status.toJSON());
+      statusYaml = YAML.stringify(tmp);
+      insertStatus(dist, tmp);
     }
   }
 
+  // Sort first by Baseline low date, TODO.
+  const compatFeatures = Array.isArray(overridden.compatFeatures) ? overridden.compatFeatures : taggedCompatFeatures;
```

Comment:
I'd like to rebase this on top of https://github.com/web-platform-dx/web-features/pull/1233 because I think it'll look nicer after that change.